CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 * Ignore unused import statements which occur in `__init__.py` (RJ722, #192).
 * Report first decorator's line number for decorated objects on Python 3.8+.
   (RJ722, #200)
+* Report issue codes in output (e.g., `code.py:1: V104 unused import ...`)
+  (RJ722, #195)
+* Support `# noqa` comments to suppress results on that line. (RJ722, #195).
```

Comment:
Could you please draft the documentation for the README.md file? It's always easier to edit existing text than writing new text.

Reply:
Sure!